ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsLocator.java

```diff
@@ -86,14 +92,16 @@
         .stream()
         .map(this::asKsqlHost);
     final Stream<KsqlHost> hostStream = Stream.concat(active, standby);
+    final List<KsqlHost> allHosts = hostStream.collect(Collectors.toList());
```

Comment:
just a single line `collect()` above , merging L94 and L95?

Reply:
Done